char_index.py

```diff
@@ -0,0 +1,29 @@
+char_position = 0
+lowecase_chars = ['countat1','a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+return_message = "Position of alphabet: {char_position}"
+character_argument_max_len = 1
+    
+def position(character_argument: str) -> str:
+    """with a letter, return its position in the alphabet
+    
+    Args:
+    String, a single character 
+    
+    Returns
+    String, a string in the format of 'Position of alphabet: char_position'
+    
+    Raises
+    ca
+    """
+    
+    # Sanity Checks
+    if not (isinstance(character_argument, str) and len(character_argument) == character_argument_max_len):
```

Comment:
checking that its a string is probably redundant if we're using and enforcing type annotations?

Reply:
How do I know if it's enforcing? My understanding of type hint is they are can be ignored?

Reply:
Typically that would be the build system that would check these they're never really enforced at runtime.